src/main/resources/kafka-topics.json

```diff
@@ -11,6 +11,10 @@
     {
       "name": "inventory.holdings-record",
       "numPartitions": 50
+    },
+    {
+      "name": "inventory.instance-contribution",
```

Comment:
What is an instance contribution? I thought the job published messages to an iteration topic?

Reply:
No, for contribution we need another, dedicated topic. Please review once again
[Initial record contribution](https://wiki.folio.org/display/DD/D2IR+Record+Contribution+flow#D2IRRecordContributionflow-Initialrecordcontribution) and [SPIKE: [MODINREACH-80]](https://wiki.folio.org/display/FOLIJET/SPIKE%3A+%5BMODINREACH-80%5D+Record+Contribution%3A+Analyze+Re-Index+job+implementation+usage+in+mod-inventory-storage)

Reply:
I don't understand. What is this topic used for?

Reply:
It's used by mod-inn-reach module to perform initial contribution of inventory to external system (Inn-Reach). Details about Initial Contribution, what it is for etc, can be found in the provided wiki pages (above)

Reply:
Ok, I don't have the capacity to do that research and review that aspect.

It seems odd to me that is part of the same work as the new iteration API. And I don't understand why either iteration or the regular events aren't sufficient for that.